libs/scraper-agent/mocked/mockAgent.ts

```
@@ -6,21 +6,23 @@ import type { NamedProcessor } from "../src/processors";
 import { SectionType, sections } from "../src/processors/types";
 import type { IAskUser, SectionName, EventSection } from "../src/types";
 import { sleep } from "@thecointech/async/sleep";
+import { AccountResponse } from "@thecointech/vqa";
 
 export class Agent implements AgentClass {
   name: string;
-  events: EventManager;
-  page: PageHandler;
   input: IAskUser;
```

ESLint

[error] Expected blank line between class members.

(lines-between-class-members)